core-helm-chart/templates/clusterrolebinding.yaml

```diff
@@ -0,0 +1,27 @@
+# Copyright 2023 The KubeStellar Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: kubestellar-clusterrole-binding
+subjects:
+- kind: ServiceAccount
+  name: kubestellar-sa
+  apiGroup: ""
+  namespace: default

```

Comment:
This is a bug since the namespace is defined by the user, use:
`namespace: {{ .Release.Namespace }}` instead

Reply:
Hey @francostellari just to make sure I am not missing anything - this should be .Values.namespace, right?  I don't see us defining a Release anywhere.